Sources/SwiftInspectorVisitors/Tests/TypeDescriptionSpec.swift

```diff
@@ -39,7 +39,7 @@ final class TypeDescriptionSpec: QuickSpec {
 
       context("when decoding a simple type") {
         beforeEach {
-          data = "{\"caseDescription\":\"simple\",\"text\":\"Foo\"}".data(using: .utf8)
+          data = "{\"caseDescription\":\"simple\",\"text\":\"Foo\",\"typeDescriptions\":[]}".data(using: .utf8)
```

Comment:
in a later commit I make this data easier to read, and in yet another later commit I add a generic type to this test.